woocommerce/wc-api-dev
# Remove `/data/counts` and `/products/reviews` endpoints

Partially fixes https://github.com/Automattic/wc-calypso-bridge/issues/49.

This PR removes the `/data/counts` and `/products/reviews` endpoints from `wc-api-dev`. These endpoints are being moved to `wc-calypso-bridge`. See https://github.com/woocommerce/woocommerce/pull/20116.

The product reviews class needs to stick around, so `/products/$product_id/reviews` still work via v3 paths (which is an existing core endpoint).

This PR also bumps the version number.

To Test:
* See testing directions on https://github.com/Automattic/wc-calypso-bridge/pull/50.
